src/Microsoft.AspNet.Authentication/ClaimsTransformationAuthenticationHandler.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
My first reaction is to question why this is modeled as an AuthenticationMiddleware. Most of the functionality of the AMW doesn't seem to be usefull. 

What about a delegate on HttpContext? It would be simpler, if all the user wants to do is perform a transformation, they must now understand the AH model.